step3/index.js

```diff
@@ -0,0 +1,103 @@
+const express = require('express')
+const app = express()
+const port = 3000
+
+const mongoose = require('mongoose')
+const config = require('./config/key') // mongoDB URI를 보호하기 위한 설정파일
+const { Item } = require('./Item')
+
+app.use(express.json()) // body-parser 대신 사용
+
+mongoose.connect(config.mongoURI, {
+    useNewUrlParser: true, useUnifiedTopology: true, useCreateIndex: true, useFindAndModify: false
+}).then(() => console.log('MongoDB Connected...'))
+    .catch(err => console.log(err))
+
+/**
+ * 아이템 조회
+ */
+app.get('/api/items', (req, res) => {
+    const items = Item.find({}, (err, items) => {
+        if(err) return res.status(500).json({success: false, err})
+        if(items.length === 0){
+            return res.json({
+                success: true,
+                message: "아이템이 없습니다."
+            })
+        }
+        res.status(200).json({success: true, message: "아이템 조회 성공", items})

```

Comment:
```suggestion
        res.status(200).json({success: true, items})
```
- message는 제거해도 무방할 것 같아요!
- 빈 배열도 값이기 때문에 반환해주면 어떨까요?